bat/tests/09-no-delta-packs-over-format-bump/description.txt

```diff
@@ -0,0 +1,5 @@
+09-no-delta-packs-over-format-bump
```

Comment:
This will clash with the other PR, can you bump the version up one more pls?

Reply:
Yeah I will when the other one gets merged.